README.adoc

```diff
@@ -521,6 +521,10 @@ class TestMeme < Minitest::Test
 end
 ----
 
+== File Naming [[file-naming]]
+
+Use a consistent naming pattern of either a `test_` prefix or a `_test` suffix for filenames of tests.
```

Comment:
I was under the impression the suffix option is way more common, but I haven't used MiniTest that much.

Reply:
Yeah, I also see the suffix option more often than the prefix option.

For example, Rails (`ActiveSupport::TestCase` inherits `Minitest::Test`) itself and Rails application template path uses the suffix option. However, Minitest itself uses the prefix option.

- Rails itself (`*_test.rb`) ... https://github.com/rails/rails/tree/master/activerecord/test/cases
- Rails app template path (`*_test.rb`) ... https://github.com/rails/rails/blob/v6.1.0.rc1/railties/lib/rails/generators/test_unit/model/model_generator.rb#L16https://github.com/rails/rails/blob/master/railties/lib/rails/generators/test_unit/model/templates/unit_test.rb.tt
- Minitest itself (`test_*.rb`) ... https://github.com/seattlerb/minitest/tree/master/test/minitest

It may be helpful to mention some use cases if the suffix option (or the prefix option) will be recommended by default.

But I'm not sure because I may just lack knowledge of plain Minitest (not `ActiveSupport::TestCase`) use cases 😅

Reply:
Thanks for the feedback. I've added a note for Rails.